### PR TITLE
Postman Test Integration

### LINKED ISSUE #18: Test for /jobs endpoint
---
Name: Test for /jobs endpoint
About: Test for /jobs endpoint to ensure it returns all fields the FE needs to consume and display. https://docs.google.com/spreadsheets/d/1mxnKK4iQ-wrJeZTF3zWcYskbkqlm4G4S5A7gXrfFMy8/edit#gid=483454443

---

### LINKED ISSUE #19: Test for /jobs/<id> endpoint
---
Name: Test for /jobs/<id> endpoint
About: Test for /jobs/<id>  endpoint to ensure it returns all fields the FE needs to consume and display. https://docs.google.com/spreadsheets/d/1mxnKK4iQ-wrJeZTF3zWcYskbkqlm4G4S5A7gXrfFMy8/edit#gid=483454443

---

### LINKED ISSUE #27: Seed command does not remove old seeds
**Describe the bug**
Need to add logic to the db seed command that drops the old seeds prior to seeding the database. Currently it adds duplicate data on top of anything in the DB

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by dtsehl**
# Description
* Add functionality to perform integration tests with Postman
* Update the Readme with new testing and Team
* Fix the bug where when the `db_seed` command is run it does not delete old seeded jobs. Now when you run the command it deletes the old seeds and makes it so newly added seeds begin with the primary key of 1.
* Even though this branch is called `dan_docker` it does not implement any Docker functionality.

# Closes issue(s)
#27 #18 #19 

# How to test / reproduce
The way to test is spelled out in the Readme:
7. run integration tests
    * First, click the button below to download the Career Day Postman collections to your local client:
    [Run in Postman](https://app.getpostman.com/run-collection/c226a3ff3a643f0456f7)
    * Next, ensure the app is running with `flask run` in the command line
    * Finally, to run the tests, in Postman:
      1. Click `Runner` at the top left
      2. In the new window that opens up, click on the Career Day collection on the left
      3. Then click the blue Run Career Day button to run all the tests
    * To examine or run just one test, in the main Postman app click on the `Tests` tab of one of the requests and click the blue `Send` button

# Screenshots

# Changes include
- [X] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist
- [X] I have tested this code
- [X] I have updated the Readme

# Other comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,16 +19,23 @@ Stack: Python3, Flask, Postgres, Heroku
...
-  ```
-  $ pytest --cov=app tests/
-  ```
+6. run model tests with coverage:
+    ```
+    $ pytest --cov=app tests/
+    ```
+7. run integration tests
+    * First, click the button below to download the Career Day Postman collections to your local client:
+    [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/c226a3ff3a643f0456f7)
```
Comment (janegreene) [inline README.md:28]: Great find, thanks for sharing this trick!

File: app.py [hunk 2]
```diff
@@ -54,6 +54,7 @@ def __repr__(self):
 
 @app.cli.command('db_seed')
 def db_seed():
+    JobsModel.query.delete()
```
Comment (janegreene) [inline app.py:57]: This clears database before reseeding.

File: app.py [hunk 3]
```diff
@@ -235,6 +236,7 @@ def db_seed():
         long_description="Pharmacists are healthcare practitioners who check and distribute drugs to doctors for medication that had been prescribed to patients. They advise patients and health care providers on the selection, dosages, interactions, and side effects of medications. They might work in a  hospital or retail store."
     )
     jobs = [job1, job2, job3, job4, job5, job6, job7, job8, job9, job10, job11, job12, job13, job14, job15]
+    db.session.execute("ALTER SEQUENCE jobs_id_seq RESTART WITH 1")
```
Comment (dtsehl) [inline app.py:239]: Nice find @Strompy on how to run raw SQL commands through SQLAlchemy!

**Review by janegreene — APPROVED** [review]
Nice documentation updates, looks great.

**Pull request merged**